.github/ISSUE_TEMPLATE/user_story.yml

```diff
@@ -0,0 +1,66 @@
+name: User Story
+description: File an issue for a new feature
+title: "[STORY]: "
```

Comment:
Question: Is the idea here that we preserve the `[Label]: ` prefixes in the title? 

This convention is pretty inconsistent across various issues today. If the idea is to promote visual consistency (make it easier to visually scan, etc.), I think it could be useful to specify that the `[STORY]: ` prefix be retained during issue creation. Unfortunately, it doesn't seem like issue templates support directly adding descriptive text around the title field, but I think some way of calling out the convention in either the `description:` field above this line or in the first `type: markdown` field would be sufficient.